design/noobaa/backing-stores.md

```diff
@@ -0,0 +1,31 @@
+♜ [Rook NooBaa Design](README.md) /
+# Backing Stores
+
+Each item in `system.spec.backingStores` will be provisioned according to its type:
+
+### Type: `aws-s3`
+
+Assume the bucket exists and set the aws credentials in NooBaa.
```

Comment:
Are the creds stored by noobaa somewhere besides the secret provided? Or they just stay in the secret provided?

Reply:
creds are stored inside the NooBaa brain PV.

Reply:
Resolving.